src/EditorFeatures/Core.Wpf/LineSeparators/LineSeparatorAdornmentManager.cs

```diff
@@ -13,50 +13,18 @@
 
 namespace Microsoft.CodeAnalysis.Editor.LineSeparators
 {
-    internal class LineSeparatorAdornmentManager : AdornmentManager<LineSeparatorTag>
+    internal class LineSeparatorAdornmentManager : AbstractAdornmentManager<LineSeparatorTag>
```

Comment:
consider pulling this out into a separate PR.

Reply:
Pull out the renaming into a separate PR?